bin/metrics-es-node-graphite.rb

```diff
@@ -264,7 +264,9 @@ def run
     node['indices'].each do |type, index|
       index.each do |k, v|
         # #YELLOW
-        unless k =~ /(_time$)/ || v =~ /\d+/
+        if k.end_with? 'is_throttled'
+          metrics["indices.#{type}.#{k}"] = v ? 1 : 0
```

Comment:
Thanks for this. I have followed your suggestions, can you please have a look at it?

Reply:
My 2 cents:
> not attempt to handle at all and bail with an ugly stack trace
Ugh, I've seen this approach taken, strongly prefer to not doing this as if we know of a failure mode. 
> catch the error and return incorrect data if so do we return true or false?
My gut says it's better to not return any data versus returning incorrect data. I'd rather someone debug why data isn't there versus using data that's not correct to make a design. 
> catch the error and abort with a meaningful message
❤️ 
> catch the error and write to STDERR rather than STDOUT...
Hmmm, this is an interesting ideahttps://github.com/sensu/sensu/blob/master/lib/sensu/client/process.rb#L167 and seem to imply that STDERR would be captured, https://github.com/sensu/sensu/blob/master/lib/sensu/client/process.rb#L198-L200 so that's not crazy in my opinion. If we do do that, I'd like to have someone quickly test what STDERR ends up looking like in the sensu event just to make sure nothing weird happens there (since I've never used that approach). 

Overall, I agree the testing for the truthiness of the value and returning it and bailing cleanly if we don't get one is probably the best move here.

Reply:
I think I am leaning towards the printing to `STDERR` unless someone has a better idea.